spec/models/admin_set_spec.rb

```diff
@@ -98,16 +98,11 @@
   end
 
   it "has a title" do
-    subject.title = ["title"]
-    subject.save
-    expect(subject.reload.title).to eq ["title"]
+    expect(subject.title).to be_a(ActiveTriples::Relation)
```

Comment:
This is a complicated one; On reflection, I think it should be two expectations. One that it is not "just an Array" but also that it has Array like behavior. Will amend.